### PR TITLE
Initial install

### PR DESCRIPTION
**Pull request by seans3**

### CODE REVIEW

File: pkg/apply/apply.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		Long:  "Declaratively create of update a resource using a config file",
+		Example: `
+		# Run apply on a config file".
+		apply -f FILENAME`,
+		Run: func(cmd *cobra.Command, args []string) {
+			err := o.Validate(cmd, args)
+			if err != nil {
+				panic(err)
+			}
+			err = o.RunApply(cmd, in, out, errOut)
```
Comment (apelisse) [inline pkg/apply/apply.go]: I really wish we didn't depend on `cmd` more than needed. That's probably one of the bad patterns that we want to get rid of.

File: cmd/apply/apply.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	"os"
+
+	"k8s.io/kubectl/pkg/apply"
+)
+
+// TestableMain allows test coverage for main.
+func TestableMain(stdin io.Reader, stdout, stderr io.Writer) error {
+	cmd := apply.NewCmdApply(stdin, stdout, stderr)
+	err := cmd.Execute()
+	if err != nil {
```
[the remark this answers is not in the document]
Comment (seans3) [reply cmd/apply/apply.go]: Fixed.

**Comment by apelisse** [comment]
Should I click a button here?

**Comment by apelisse** [comment]
We probably don't want this anymore, do we?

**Pull request closed without merging**